collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -447,13 +449,16 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     String accountName = data.getStringExtra(AccountManager.KEY_ACCOUNT_NAME);
                     accountsManager.setSelectedAccountName(accountName);
                 }
+                selectedGoogleAccountPreference.setEnabled(true);
                 break;
+
         }
     }
 
     @Override
     public void onGoogleAccountSelected(String accountName) {
         selectedGoogleAccountPreference.setSummary(accountName);
+        selectedGoogleAccountPreference.setEnabled(true);
```

Comment:
Why do we need that change here as well? I thought that in `onActivityResult` would be enough. Am I wrong?